src/pybind/mgr/mgr_module.py

```diff
@@ -413,13 +413,32 @@ class CPlusPlusHandler(logging.Handler):
     def __init__(self, module_inst):
         super(CPlusPlusHandler, self).__init__()
         self._module = module_inst
-        self.setFormatter(logging.Formatter("[{}] [%(levelname)-4s] [%(name)s] %(message)s"
+        self.setFormatter(logging.Formatter("[{} %(levelname)-4s %(name)s] %(message)s"
                           .format(module_inst.module_name)))
 
     def emit(self, record):
         if record.levelno >= self.level:
             self._module._ceph_log(self.format(record))
 
+class ClusterLogHandler(logging.Handler):
+    def __init__(self, module_inst):
+        super(ClusterLogHandler, self).__init__()
+        self._module = module_inst
+        self.setFormatter(logging.Formatter("%(message)s"))
```

Comment:
If we prefer other python string formatter rather than `%` syntax, here [we have the chance to use another](https://docs.python.org/3/library/logging.html#logging.Formatter).
```py
self.setFormatter(logging.Formatter("{message}", style='{'))
```